src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+interface Props {
+  people: Person[]
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { personSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map((person) => {
+          const {
+            name,
+            sex,
+            born,
+            died,
+            motherName,
+            mother,
+            fatherName,
+            father,
+            slug,
+          } = person;
+
+          return (
+            <tr
+              key={name}
+              data-cy="person"
+              className={classNames({
+                'has-background-warning': personSlug === slug,
+              })}
+            >
+              <td>
+                <PersonLink person={person} />
+              </td>
+
+              <td>{sex}</td>
+              <td>{born}</td>
+              <td>{died}</td>
+              <td>
+                { mother
+                  ? <PersonLink person={mother} />
+                  : motherName || '-'}
```

Comment:
ternary operator should return one value depending on condition, don't make another condition in the ternary operator